lib/offline.js

```diff
@@ -136,7 +136,14 @@ module.exports = function(config) {
 
     function checkImports(imports, rootDir) {
       imports.forEach(function (filepath) {
-        assertExists(path.join(rootDir, filepath));
+        try {
+          assertExists(path.join(rootDir, filepath));
+        } catch (ex) {
+          process.stdout.write(chalk.yellow.bold(
+            '⚠ ' + filepath + ' doesn\'t exist or is a directory, but you\'ve added it to importScripts.\n' +
+            '  Are you sure you want to import it?\n'
```

Comment:
This asks a question but doesn't prompt the user to answer it. I wouldn't actually prompt here, since the user may have a good reason for doing this (f.e. they intend to create the file but haven't done so yet). And it's a warning, not an error (thus doesn't need to be corrected). But I'd either direct the user to address the problem:

>⚠ file/path doesn't exist or is a directory, but you've added it to importScripts.
>  To fix this problem, ensure file/path is a file or remove it from importScripts.

Or simply note the problem without any direction:

>⚠ file/path doesn't exist or is a directory, but you've added it to importScripts.